cmd/sonobuoy/app/gen_plugin.go

```diff
@@ -1,127 +0,0 @@
-/*
```

Comment:
Should be able to rebase onto `upstream/master` and this commit shouldn't come up anymore.